docs/implementation-guide.md

```diff
@@ -55,13 +51,12 @@ We have only provided a brief overview of PHDI, Building Blocks, and the pipelin
 ## Implementing the PHDI Azure Pipelines
 In this section we describe how a STLT can take this repository and use it to spin up all of the functionality that it offers in their own Azure environment.
 
-### User Assumptions
-In order to proceed you will need either:
-- an Azure account with permissions to create new resource groups in your organization's Azure environment,
-or
-- an Azure account with `Owner` access to a resource group in your organizations’ Azure environment that was created to house the PHDI Azure pipelines, and the name of this resource group.
+### User Requirements
+In order to proceed you will need:
+1. `Owner` access to the Azure subscription where you would like to deploy the PHDI Starter Kit.
+2. `Admin` access to the GitHub account or organization where your copy of this repository will be created.
```

Comment:
I'm not sure this is true - I think you just need enough permissions to be able to create repositories in the organization.

Reply:
Good point!